h2o-core/src/main/java/water/api/schemas3/FramesV3.java

```diff
@@ -43,6 +43,9 @@ public class FramesV3 extends RequestSchemaV3<Frames, FramesV3> {
   @API(help="Use parallel export to a single file (doesn't apply when num_parts != 1, creates temporary files in the destination directory)",json=false)
   public boolean parallel;
 
+  @API(help="Output file format. One of 'csv', 'parquet'. Defaults to 'parquet'.",json=false)
```

Comment:
It might be a silly question but wouldn't it be better to have the `format` as `enum` and in the API have the values specified? E.g.
```
@API(help="Output file format. One of 'csv', 'parquet'. Defaults to 'parquet'.", values={"csv", "parquet"} , json=false)
```

If so, the same would apply for the `compression` field below.

Reply:
yes yes, will change to enum 👍  Not sure about compression regarding backwards compatibility